packages/dev/core/src/Shaders/ShadersInclude/pbrBlockPrePass.fx

```diff
@@ -0,0 +1,88 @@
+float writeGeometryInfo = finalColor.a > ALPHATESTVALUE ? 1.0 : 0.0;

```

Comment:
just confirming you tested that change with NME as well

Reply:
This has no impact on the `PrePassOutputBlock` block, as the code is generated by the block, according to the inputs connected.